tests/nvmeof/workflows/initiator.py

```diff
@@ -14,6 +14,32 @@ def __init__(self, node, gateway):
         self.gateway = gateway
         self.discovery_port = 8009
 
+    def fetch_nvme_devices_for_gw(self, gateway):
+        """Fetch nsids for all devices optimised for the given gateway
+
+        Returns:
+            nsids for which the given gateway is optimised.
+            empty list if the gateway is inaccessible for all devices
+        """
+        out, _ = self.node.exec_command(
+            cmd="nvme list --output-format json", sudo=True
+        )
+        nvme_devices = json.loads(out).get("Devices", [])
```

Comment:
At client side,
- Need to identify the right devices against the namespace passed from GW side(probably using lsblk -o  name,wwn). 
- Get the topology for that device(i.e get the path status for each GW path).
- Validate against the GW which has optimized path.

Reply:
nvme list-subsys gives the namespace id and subsystem name, isn't this enough proof to get the required namespaces?

Reply:
@sunilkumarn417 nvme show-topology is useful only when we are trying to get ana states for all namespaces at once. its not useful in our approach where we are going with per device validation. So I will retain nvme list-subsys itself for now.
Ref - https://github.com/linux-nvme/nvme-cli/issues/1544